### PR TITLE
[HotFix] Fix build issue after #2286

### PR DESCRIPTION
**Pull request by averinevg**
Compilation has been broken by merging https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2295 and https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2286

### CODE REVIEW

**Review by iq136boy — APPROVED** [review]

**Comment by atamazov** [comment]
@junliume @JehandadKhan Please merge this ASAP to unblock CI.

**Comment by JehandadKhan** [comment]
@atamazov  Waiting on the CI to pass.

**Review by junliume — APPROVED** [review]

**Comment by junliume** [comment]
It passed the stage where it failed before, thanks for the quick catch!

**Pull request merged**
